_episodes/12-cluster.md

```diff
@@ -162,19 +135,18 @@ notice that the current hostname is also part of our prompt!)
 Individual computers that compose a cluster are typically called *nodes* (although you will also
 hear people call them *servers*, *computers* and *machines*). On a cluster, there are different
 types of nodes for different types of tasks. The node where you are right now is called the *head
-node*, *login node* or *submit node*. A login node serves as an access point to the cluster. As a
-gateway, it is well suited for uploading and downloading files, setting up software, and running
-quick tests. It should never be used for doing actual work.
+node*, *login node*, *landing pad*, or *submit node*. A login node serves as an access point to the
+cluster. As a gateway, it is well suited for uploading and downloading files, setting up software,
```

Comment:
`As a gateway, it is well suited for uploading and downloading files` not sure this is true while reading it. I propose:

```suggestion
cluster. As a gateway, it is well suited for uploading and downloading small files, setting up software,
```

Shall we fix it right now?

Reply:
I don't think the "small" qualifier helps to clarify anything; if anything, it opens a discussion, "how small is small," that would be distracting. Practically, the limit is your filesystem quota, and whether your bandwidth utilization impacts other users.

We could open a separate issue to call out data transfer nodes, which are sometimes available for large (high-bandwidth) data transfers?

Reply:
I like the idea with a call-out on this!